### PR TITLE
add face demo and fix heads

### PR DESCRIPTION
**Pull request by jin-s13**

### CODE REVIEW

**Comment by innerlee** [comment]
Add face dataset to repo's readme btw

File: mmpose/apis/inference.py [hunk 1]
```diff
@@ -208,6 +208,28 @@ def _inference_single_pose_model(model,
...
+
+    elif dataset in 'FaceCOFWDataset':
+        flip_pairs = [[0, 1], [4, 6], [2, 3], [5, 7], [8, 9], [10, 11],
+                      [12, 14], [16, 17], [13, 15], [18, 19], [22, 23]]
+
+    elif dataset in 'FaceWFLWDataset':
+        flip_pairs = [[0, 32], [1, 31], [2, 30], [3, 29], [4, 28], [5, 27],
+                      [6, 26], [7, 25], [8, 24], [9, 23], [10, 22], [11, 21],
+                      [12, 20], [13, 19], [14, 18], [15, 17], [33,
+                                                               46], [34, 45],
```
Comment (innerlee) [inline mmpose/apis/inference.py:224]: The formatting is so odd. Must be bugs in yapf

File: README.md [hunk 2]
```diff
@@ -67,11 +67,12 @@ Supported backbones for human pose estimation:
 - [x] [ShufflenetV1](configs/top_down/shufflenet_v1/README.md)
 - [x] [ShufflenetV2](configs/top_down/shufflenet_v2/README.md)
 
-Supported methods for human pose estimation:
+Supported methods:
```
Comment (innerlee) [inline README.md:70]: algorithms

**Review by innerlee — APPROVED** [review]

**Comment by innerlee** [comment]
Do you want to add something to the demo/ folder?

**Comment by jin-s13** [comment]
> Do you want to add something to the demo/ folder?

Not now. Need to train a face detector using MMDet before that.

**Pull request merged**